src/cdk/schematics/update-tool/utils/imports.ts

```diff
@@ -8,39 +8,106 @@
 
 import * as ts from 'typescript';
 
-export type Import = {
-  name: string,
-  importModule: string,
-  node: ts.ImportDeclaration
-};
-
-/** Gets import information about the specified identifier by using the type checker. */
-export function getImportOfIdentifier(typeChecker: ts.TypeChecker, node: ts.Identifier): Import|
+/** Interface describing a resolved import. */
+export interface Import {
+  /** Name of the imported symbol. */
+  symbolName: string;
+  /** Module name from which the symbol has been imported. */
+  moduleName: string;
+}
+
+
+/** Resolves the import of the specified identifier. */
+export function getImportOfIdentifier(node: ts.Identifier, typeChecker: ts.TypeChecker): Import|
     null {
-  const symbol = typeChecker.getSymbolAtLocation(node);
+  const directImport = getSpecificImportOfIdentifier(node, typeChecker);
```

Comment:
Add some comments here with examples of the different cases (e.g. a "direct import" vs. a "qualified import")?

Reply:
Added.